specification/sql/resource-manager/Microsoft.Sql/preview/2018-06-01-preview/managedInstanceSensitivityLabels.json

```diff
@@ -0,0 +1,523 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2018-06-01-preview",
+    "title": "SqlManagementClient",
+    "description": "The Azure SQL Database management API provides a RESTful set of web APIs that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases, servers, and other entities."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/managedInstances/{managedInstanceName}/databases/{databaseName}/schemas/{schemaName}/tables/{tableName}/columns/{columnName}/sensitivityLabels/{sensitivityLabelSource}": {
```

Comment:
schema, tables and columns are not modeled through the RP control plane. And you are exposing a nested resource type under data plane entities. This can potentially create a  gap in the experience. Can these three be modeled as properties you take as input on create instead of having them in the path? For GET, if user wants to get sensitivityLabel for a specific schema/table/column, they can use filter query param?
And then you can model sensitivityLabel directly under the database resource?

Reply:
Chatted with Igal and although it doesnt seem ideal but since other existing APIs are already using similar pattern, I will sign off on this. IN the meantime, Igal will discuss within his team if they would like to re-design these for a newer api-version, or GA version.